spec/api/work_package_resource_spec.rb

```diff
@@ -76,4 +77,34 @@
       it { expect(parsed_response['remainingHours']).to be_nil }
     end
   end
+
+  describe '#patch' do
+    let(:valid_params) do
+      {
+          _type: 'WorkPackage',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.